static/src/reducers/paths.js

```diff
@@ -2,7 +2,8 @@ import { default as UUID } from 'node-uuid';
 import { indexElementWithId } from '../utils/arrayUtils';
 import { ADD_PATH, SET_PATH_SYNC, STARTSTOP_PATH,
   SET_PATH_CURRENTLY_PLAYING, SELECT_PATH, DELETE_SOUND_FROM_PATH,
-  ADD_SOUND_TO_PATH, CLEAR_ALL_PATHS, SET_PATH_WAIT_UNTIL_FINISHED } from '../actions/actionTypes';
+  ADD_SOUND_TO_PATH, CLEAR_ALL_PATHS, SET_PATH_WAIT_UNTIL_FINISHED,
+  SET_PATH_ACTIVE } from '../actions/actionTypes';
 
 const initialState = {
   paths: [],
```

Comment:
I would consider having two separate reducers, `paths` and `selectedPath`, and then export `combineReducers({ paths, selectedPath })`. Would make the code much more scalable!